### PR TITLE
Style invest summary page.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Makes the invest summary page responsive
Screen Shot 2022-01-25 at 15 48 35
Screen Shot 2022-01-25 at 15 48 06

### CODE REVIEW

**Comment by W3stside** [comment]
point this to `release/1.10`

**Comment by elena-zh** [comment]
Changes LGTM besides 2 tiny issues: 
1. 'Connected account' labels seem like shifted to the bottom
shifted

2. Investment amount is also shifted towards fields' labels, other values:
inv amount is shifted

**Review by alfetopito — APPROVED** [review]
~Looks good, and I also second Elena on the stepper on mobile, which you might address in another PR?~

Comment meant for another PR. Will review this one now and post a relevant comment :)

**Review by alfetopito — APPROVED** [review]
Yes, looks great 👍

**Review by anxolin — APPROVED** [review]
Nice!

ubernit: maybe im wrong, but it sounds like a better layout with the select inline with the content:

image

**Comment by biocom** [comment]
> Nice!
> 
> ubernit: maybe im wrong, but it sounds like a better layout with the select inline with the content:
> 
> image

Could be an option as well. But occupies vertical space (which could be used by the other elements instead). Particularly important on smaller screen devices. Open to modify/review in a separate PR to keep this going.

File: src/custom/pages/Claim/InvestmentFlow/InvestSummaryRow.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function InvestSummaryRow(props: Props): JSX.Element | null {
 
   return (
     <tr>
-      <td>
+      <td data-title="Type of Claim">
```
[the remark this answers is not in the document]
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/InvestSummaryRow.tsx:25]: On desktop the headings come from the regular `<th>` table heading elements. On mobile responsive they are of no use. Without introducing new DOM elements just for the mobile/responsive headings (they need to live in a specific DOM tree position), adding data-attributes is IMO a cleaner solution.

**Pull request merged**